e2e-tests/gef/cypress.config.js

```diff
@@ -24,4 +24,5 @@ module.exports = defineConfig({
     baseUrl: 'http://localhost',
     specPattern: 'cypress/e2e/**/*.spec.js',
   },
+  experimentalCspAllowList: ['child-src', 'default-src', 'frame-src', 'form-action', 'script-src', 'script-src-elem'],
```

Comment:
If we're trying to allow as many csp headers as possible -- could we just set `experimentalCspAllowList: true` ?

Reply:
That's what you'd expect but that's not the way it works! As per https://docs.cypress.io/guides/references/experiments#Strip-Minimum-CSP-Directives, the way it works is:
- `false`: the whole CSP header gets stripped
- `true`:
   - strips the CSP directives that _always_ break Cypress
   - strips the CSP directives that _conditionally_ break Cypress
   - leaves the CSP directives that don't affect Cypress alone
- an array of directives: strips the CSP directives that _always_ break Cypress
   - strips the CSP directives that _always_ break Cypress
   - strips any CSP directives that _conditionally_ break Cypress that you *do not* list
   - leaves any CSP directives that _conditionally_ break Cypress that you *do* list
   - leaves the CSP directives that don't affect Cypress alone